src/NexusMods.App/TelemetryProvider.cs

```diff
@@ -35,6 +37,30 @@ public TelemetryProvider(IServiceProvider serviceProvider)
             .QueryWhenChanged(datoms => datoms.Sum(d => d.V))
             .SubscribeWithErrorLogging(totalDownloadSize => _downloadSize = Size.From((ulong)totalDownloadSize))
             .DisposeWith(_disposable);
+
+        Loadout
+            .ObserveAll(_connection)
+            .FilterImmutable(static loadout => loadout.IsVisible())
+            .TransformOnObservable(loadout => LoadoutDataProviderHelper.CountAllLoadoutItems(serviceProvider, loadout.LoadoutId))
+            .QueryWhenChanged(query =>
+            {
+                _loadoutCounts.Clear();
+
+                foreach (var kv in query.KeyValues)
+                {
+                    var loadout = Loadout.Load(_connection.Db, kv.Key);
+                    if (!loadout.IsValid()) continue;
+
+                    var gameName = loadout.Installation.Name;
+                    _loadoutCounts.TryGetValue(gameName, out var counts);
+
+                    _loadoutCounts[gameName] = counts + kv.Value;
+                }
+
+                return 0;
+            })
+            .SubscribeWithErrorLogging()
```

Comment:
This is grouping by game install not by loadout. This is counting all the mods for a specific game, not all the mods for a loadout. 
Should we return the average number of mods per loadout per game? The max?

Reply:
Updated to use original behavior.